src/gui_common/TweakedColourPicker.cs

```diff
@@ -378,6 +401,21 @@ private void OnRawButtonToggled(bool isOn)
 
     private void OnColourChanged(Color colour)
     {
+        // Hide HtmlColourEdit color change when Raw mode is on and any color value is above 1.0
+        if (mode == PickerMode.Raw)
+        {
+            if (colour.IsRaw())
+            {
+                htmlColourStart.Visible = false;
+                htmlColourEdit.Visible = false;
+            }
+            else
+            {
+                htmlColourStart.Visible = true;
+                htmlColourEdit.Visible = true;
+            }
+        }
```

Comment:
```c#
htmlColourStart.Visible = !(mode == PickerMode.Raw && colour.IsRaw());
htmlColourEdit.Visible = !(mode == PickerMode.Raw && colour.IsRaw());
```

Reply:
Would this be logically correct?

Reply:
htmlColourStart.Visible = htmlColourEdit.Visible = !(mode == PickerMode.Raw && colour.IsRaw());

Reply:
I think it's right.

Reply:
And also solves a potential bug